pkg/app/image_prune.go

```diff
@@ -0,0 +1,68 @@
+package app
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	dockertypes "github.com/docker/docker/api/types"
+	"github.com/docker/go-units"
+	"github.com/olekukonko/tablewriter"
+	"github.com/urfave/cli/v2"
+
+	"github.com/tensorchord/envd/pkg/envd"
+)
+
+var CommandPruneImages = &cli.Command{
+	Name:   "prune",
+	Usage:  "Remove unused images",
+	Action: pruneImages,
+}
+
+func pruneImages(clicontext *cli.Context) error {
+
+	envdEngine, err := envd.New(clicontext.Context, "docker")
```

Comment:
We refactored the interface, maybe you can use github.com/tensorchord/envd/pkg/docker directly.

Reply:
ok i just found out. I've already changed it.

Reply:
There are some other issues here, I found that docker's prune has some args, do we need to support？